README.md

```diff
@@ -120,7 +125,7 @@ Which fragment losses to consider in the search.
 Here, additional configurations may be set using the text syntax as in the advanced config or in a config file used by the command line version of xiSEARCH (see next section)
 
 ### Do FDR setting 
-If the "Do FDR" box is ticked, xiFDR will automatically be run at the end of xiSEARCH. We tend to leave this option off, as we prefer to run xiFDR in a standalone process to have access to more advanced FDR filtering options.
+If the "Do FDR" box is ticked, xiFDR will automatically be run at the end of xiSEARCH. We tend to leave this option off, as we prefer to run [xiFDR](https://github.com/Rappsilber-Laboratory/xiFDR) in a standalone process to have access to more advanced FDR filtering options.
```

Comment:
```suggestion
If the "Do FDR" box is ticked, xiFDR will automatically be run at the end of xiSEARCH. We tend to leave this option off, as we prefer to run [xiFDR](https://github.com/Rappsilber-Laboratory/xiFDR) in a standalone process to have access to more advanced FDR filtering options. Alternative is to tick the "GUI" checkbox. This will start xiFDR for the generated output directly - but a s a independent GUI. where you can then define all parameters. It will be preconfigured to some extend - but you still need to read in the result file (basically just press "read")
```

Reply:
"extend" should be "extent". "basically just" can be removed.
"In the GUI, results should be read in by pressing the "read" button" or something like that

Reply:
Updated